src/modules/dashboard/components/ColonyHomeActions/ColonyHomeActions.tsx

```diff
@@ -227,15 +228,15 @@ const ColonyHomeActions = ({ colony, ethDomainId }: Props) => {
       },
     },
     // @todo - ready to be implemented in another PR.
-    // {
-    //   component: AddExistingSafeDialog,
-    //   props: {
-    //     prevStep: 'dashboard.AddExistingSafeDialog',
-    //     colony,
-    //     isVotingExtensionEnabled,
-    //     ethDomainId,
-    //   },
-    // },
+    {
+      component: AddExistingSafeDialog,
+      props: {
+        prevStep: 'dashboard.ManageGnosisSafeDialog',
```

Comment:
Why the change from Add to Manage ?

Are we planning on reusing this modal later for another Gnosis action ?

Reply:
This is defining the prevStep, so, this would be correct, right?